static/redfish/v1/JsonSchemas/PowerSupply/PowerSupply.json

```diff
@@ -555,4 +555,4 @@
     "owningEntity": "DMTF",
     "release": "2021.1",
     "title": "#PowerSupply.v1_1_0.PowerSupply"
-}
```

Comment:
Why delete it and add it again?

Reply:
I don't know why it is showing really.. but , everything seems be OK to me..